pki/util.go

```diff
@@ -342,7 +374,17 @@ func getEtcdCertKeys(rkeNodes []v3.RKEConfigNode, etcdRole string) []string {
 	}
 	etcdHosts := hosts.NodesToHosts(rkeNodes, etcdRole)
 	for _, host := range etcdHosts {
-		certList = append(certList, GetEtcdCrtName(host.InternalAddress))
+		certList = append(certList, GetCrtNameForAddress(host.InternalAddress, EtcdCertName))
+	}
+	certList = append(certList, getKubeletCertKeys(rkeNodes)...)
+	return certList
+}
+
+func getKubeletCertKeys(rkeNodes []v3.RKEConfigNode) []string {
```

Comment:
can we make a generic method to get cert/keys for specific hosts, you can pass the nodes + the role and return the certlist